pkg/errors/errors.go

```diff
@@ -101,5 +92,10 @@ func New(errString string) error {
 }
 
 func NewWMIError(errorCode uint16) error {
```

Comment:
In addition to NewWMIError, let's add a "func NewWMIErrorString" function that would do the following:

func NewWMIError(errorString string) error {
    return fmt.Error(errorString)
}